NitroxClient/GameLogic/Helper/InventoryContainerHelper.cs

```diff
@@ -55,8 +55,15 @@ public static bool TryGetOwnerId(Transform ownerTransform, out NitroxId ownerId)
                 ownerId = null;
                 return false;
             }
+            // TODO: in the future maybe use a switch on the PrefabId (it's always the same structure in a prefab)
+            // and then statically look for the right object because we'll know exactly which one it is
 
-            if (parent.GetComponent<Constructable>() || parent.GetComponent<IBaseModule>().AliveOrNull())
+            // This check must happen before the GetComponent<IBaseModule> which will be triggered (wrong result)
```

Comment:
I'm unclear what this is trying to say. Is it saying that if the GetComponent call is done first it will produce an unwanted outcome? If so I think the end of the comment should be reworded it's a little unclear if being triggered is wrong or if this check prevents that